pkg/build/api/types.go

```diff
@@ -199,6 +199,9 @@ type STIBuildStrategy struct {
 	// Image is the image used to execute the build.
 	Image string `json:"image,omitempty"`
 
+	// Additional environment variables you want to pass into a builder container
+	Env []kapi.EnvVar `json:"env,omitempty"`
```

Comment:
@smarterclayton CustomBuild does define Env, so first thing would be
consistency. Also you might have override some Env vars for the STI that
should be passed down to the assemble scripts.
On Jan 29, 2015 10:14 PM, "Clayton Coleman" <notifications@github.com>
wrote:

> In pkg/build/api/types.go
> <https://github.com/openshift/origin/pull/785#discussion_r23803112>:
>
> > @@ -199,6 +199,9 @@ type STIBuildStrategy struct {
> >  	// Image is the image used to execute the build.
> >  	Image string `json:"image,omitempty"`
> >
> > +	// Additional environment variables you want to pass into a builder container
> > +	Env []kapi.EnvVar `json:"env,omitempty"`
>
> Is there a story backing this or an end user use case?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/785/files#r23803112>.
>